core/src/main/java/com/scalar/db/storage/cosmos/CosmosUtils.java

```diff
@@ -0,0 +1,7 @@
+package com.scalar.db.storage.cosmos;
+
+public final class CosmosUtils {
```

Comment:
Please don't just ignore my suggestion. If you have reasons why you don't want to apply my suggestion, please tell me and we can discuss it:
```suggestion
public final class CosmosUtils {

  private CosmosUtils() {}

```

Reply:
Ah sorry, I missed the method.

Reply:
> Why do we need this method btw?

I found a good article for it. You can check it:
https://www.informit.com/articles/article.aspx?p=1216151&seqNum=4

Reply:
Great! Thank you!